x-pack/plugins/security_solution/public/common/lib/telemetry/events/attack_discovery/index.ts

```diff
@@ -18,6 +18,27 @@ export const insightsGeneratedEvent: TelemetryEvent = {
         optional: false,
       },
     },
+    durationMs: {
+      type: 'integer',
+      _meta: {
+        description: 'Duration of request in ms',
+        optional: false,
+      },
+    },
+    alertsCount: {
+      type: 'integer',
+      _meta: {
+        description: 'Number of alerts evaluated',
```

Comment:
nit: Consider something like `Number of unique alerts referenced in the attack discoveries`, because this may be less than the number of alerts evaluated.